packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1694,7 +1695,7 @@ export class IntervalCollection
 		}
 
 		const { id } = getSerializedProperties(serializedInterval);
-		const interval = this.localCollection.idIntervalIndex.getIntervalById(id);
+		const interval = this.getIntervalById(id);
```

Comment:
feels like we should probably dispose in this function after emitting a change to the application

Reply:
That might add some minor inconsistencies around disposing intervals created while detached but since all it does for now is give us clearer errors, im ok with that.

Reply:
i poked around here a bit and couldn't find a cheap solution. it also starts to bleed into undo redo which reuses instances.